ios/mapswipe/Info.plist

```diff
@@ -39,6 +39,10 @@
 	</dict>
 	<key>NSLocationAlwaysUsageDescription</key>
 	<string>MapSwipe doesn't actually use your location.</string>
+	<key>GOOGLE_ANALYTICS_IDFV_COLLECTION_ENABLED</key>
+	<false/>
+	<key>GOOGLE_ANALYTICS_DEFAULT_ALLOW_AD_PERSONALIZATION_SIGNALS</key>
+	<false/>
```

Comment:
for android you disable three things, here only two. Is there a particular reason for this?

Is it because the collect different things on Android '(adid, ssaid) and iOS (idfv, ad personlization signals)?

Reply:
we removed IDFA already for iOS